test/sse_server_test.exs

```diff
@@ -136,4 +140,75 @@ defmodule SSETestServerTest.SSEServerTest do
     :ok = SSEServer.end_stream(pid, "/events")
     assert_events(task, [:keepalive, {"myevent", "mydata"}])
   end
+
+  @tag :http
```

Comment:
The name `http` might be a bit broad since there are other HTTP things :/ `http_post`, `http_api`?

Reply:
`http_api` seems reasonable. I mostly just added the `http` tag so I wouldn't have to run all the other tests while I was working specifically on the HTTP API stuff.